### DISCUSSION TITLE
Will nuxt.js be updated and adjusted to vue3?

### DISCUSSION THREAD

**Discussion by spelling203** [nuxt/nuxt#9434] [Kind words]
Hi, Just wandering if there are any plans to upgrade to Vue 3?

**Comment by TheAlexLichter** [comment]
Yes. Please follow along https://github.com/nuxt/nuxt.js/issues/5708 as well as https://twitter.com/nuxt_js ☺️

**Reply by advaiyalad** [reply]
@manniL It's been a while since there've been any updates on #5708. Are there any new updates?

**Comment by agnelnieves** [comment]
Take a look at the new nuxt preview website, on the nuxt 3 announcement page. They talk about the new features.

usr: nuxt
pw: nuxt

So basically, yes, it's still on the works but it's definitely coming up on nuxt3.

Nuxt new website preview was originally posted on: #9535
